src/Shared/EventArgsFormatting.cs

```diff
@@ -181,6 +191,11 @@ int threadId
             }
             */
 
+            if (target_framework != null)
+            {
+                format.Append("TargetFramework: " + target_framework + " ");
```

Comment:
I think this creates each string individually, then concatenates them together into a new string, then adds it to the StringBuilder. It's more efficient to append multiple times.